### PR TITLE
Revert "calibrate head camera"

### PR DESCRIPTION
**Pull request by NadiaHG**
Reverts ipa320/cob_calibration_data#123

### CODE REVIEW

**Comment by fmessmer** [comment]
Why revert?....just re-calibrate and send a new PR with the new parameters - without reverting...

https://github.com/ipa320/cob_robots/pull/629 has been merged btw

File: cob4-5/calibration/cameras/head_cam.yaml [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-  cols: 5
-  data: [-0.673240, -0.518701, 0.031812, 0.053300, 0.000000]
-rectification_matrix:
-  rows: 3
-  cols: 3
-  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
-projection_matrix:
-  rows: 3
-  cols: 4
-  data: [-665.389587, 0.000000, 1402.237218, 0.000000, 0.000000 -6154.207520, 1537.669129, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
```
Comment (NadiaHG) [inline cob4-5/calibration/cameras/head_cam.yaml:20]: a comma is missed, with this version you can't start the camera driver

**Comment by NadiaHG** [comment]
a comma is missed, with this version you can't start the camera driver. And also with the comma , the calibration is wrong

**Comment by fmessmer** [comment]
closed as #126 was accepted

**Pull request closed without merging**